.idea/modules.xml

```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/PixelPerfect.iml" filepath="$PROJECT_DIR$/PixelPerfect.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/pixelperfect-client.iml" filepath="$PROJECT_DIR$/pixelperfect-client.iml" />
```

Comment:
same as 2 lines above, except my project is localy named  'pixelperfect-client' instead of 'PixelPerfect'. Dont think this breaks it, but can remove it if needed.